setup.py

```diff
@@ -3,12 +3,17 @@
 import os
 import sys
 
-from Cython.Distutils import build_ext
 from distutils.sysconfig import get_config_var
 import numpy as np
 from setuptools import setup, Extension, find_packages
 import versioneer
 
+# Default to using cython, but use the .c files if it doesn't exist
+try:
+    from Cython.Build import cythonize
+    USE_CYTHON = True
+except ImportError:
+    USE_CYTHON = False
```

Comment:
Could we ditch the `USE_CYTHON` and opt for
```python
try:
    from Cython.Build import cythonize
except ImportError:
    cythonize = False
```

Then substitute the use of `USE_CYTHON` with `cythonize`...